qiskit/providers/aer/noise/noise_model.py

```diff
@@ -299,15 +305,24 @@ def from_backend(cls, backend,
         if isinstance(backend, (BaseBackend, Backend)):
             properties = backend.properties()
             basis_gates = backend.configuration().basis_gates
+            num_qubits = backend.configuration().num_qubits
+            dt = backend.configuration().dt
             if not properties:
                 raise NoiseError('Qiskit backend {} does not have a '
                                  'BackendProperties'.format(backend))
         elif isinstance(backend, BackendProperties):
+            warn(
+                'Accepting BackendProperties object for "backend" option'
+                ' has been deprecated as of qiskit-aer 0.10.0 and'
+                ' will be removed no earlier than 3 months from that release date.',
+                DeprecationWarning, stacklevel=2)
             properties = backend
             basis_gates = set()
             for prop in properties.gates:
                 basis_gates.add(prop.gate)
             basis_gates = list(basis_gates)
+            num_qubits = len(properties.qubits)
+            dt = 1  # dummy value (may cause unrealistic noises)
```

Comment:
I think it would be better to set `dt=0` and disable delay noise and update the deprecation warning to add to the above warning that "duration dependent delay noise on scheduled circuits is disabled when initialized with backend properties".
```suggestion
            dt = 0 # disable delay noise if dt is unknown
```